.github/workflows/lint.yml

```diff
@@ -0,0 +1,11 @@
+name: lint & build
+on: push
+
+jobs:
+  lint-build:
+    name: lint & build
+    runs-on: ubuntu-latest
+    steps:
```

Comment:
maybe you also need a setup go step here

Reply:
added that step :)